src/main/kotlin/com/terraformation/backend/accelerator/api/DeliverablesController.kt

```diff
@@ -0,0 +1,206 @@
+                category = DeliverableCategory.LegalEligibility,
+                descriptionHtml = "<p>A description</p>",
+                id = DeliverableId(1),
+                name = "Incorporation Documents",
+                numDocuments = 2,
+                organizationId = OrganizationId(1),
+                organizationName = "Test Org",
+                participantId = ParticipantId(2),
+                participantName = "Random Participant",
+                projectId = ProjectId(3),
+                projectName = "Omega Project",
+                status = SubmissionStatus.Rejected,
+                type = DeliverableType.Document,
+            ),
+            ListDeliverablesElement(
+                category = DeliverableCategory.FinancialViability,
+                descriptionHtml = "<p>All about money!</p>",
+                id = DeliverableId(1),
+                name = "Budget",
+                numDocuments = 0,
+                organizationId = OrganizationId(1),
+                organizationName = "Test Org",
+                participantId = ParticipantId(2),
+                participantName = "Random Participant",
+                projectId = ProjectId(3),
+                projectName = "Omega Project",
+                status = SubmissionStatus.NotSubmitted,
+                type = DeliverableType.Document,
+            ),
+        ))
+  }
+
+  @ApiResponse200
+  @ApiResponse404
+  @GetMapping("/{deliverableId}")
+  @Operation(
+      summary = "Gets the details of a single deliverable and its submission documents, if any.")
+  fun getDeliverable(@PathVariable deliverableId: DeliverableId): GetDeliverableResponsePayload {
+    return GetDeliverableResponsePayload(
+        DeliverablePayload(
+            category = DeliverableCategory.LegalEligibility,
+            descriptionHtml = "<p>A description</p>",
+            documents =
+                listOf(
+                    SubmissionDocumentPayload(
+                        Instant.now(),
+                        "Project's articles of incorporation",
+                        DocumentStore.Google,
+                        SubmissionDocumentId(13974),
+                        "Incorporation Documents_2024-02-28_Omega_Projects articles of incorporation.doc",
+                    )),
+            feedback = "Is this a joke? This is a bunch of cat photos, not a legal document.",
+            id = deliverableId,
+            internalComment = "These guys are real jokers.",
+            name = "Incorporation Documents",
+            organizationId = OrganizationId(1),
+            organizationName = "Test Org",
+            participantId = ParticipantId(2),
+            participantName = "Random Participant",
+            projectId = ProjectId(3),
+            projectName = "Omega Project",
+            status = SubmissionStatus.Rejected,
+            type = DeliverableType.Document,
+        ))
+  }
+
+  @ApiResponse(
+      responseCode = "307",
```

Comment:
@karthikbtf @nickgraz Would it be better for this to return HTTP 200 with a response body that includes the target URL, rather than returning a redirect?

Reply:
Redirect sounds good. Ideally client would generate an anchor tag with href pointing to `/api/v1/accelerator/deliverables/{deliverableId}/documents/{documentId}` and target = '_blank'. With a 200 / body with target url, it would be extra steps on the client side to generate the href. Nick, thoughts?

Reply:
I think this is good as-is, on the frontend we will just have this URL open in a new window so the browser will follow the redirect no problem.

Reply:
You beat me to it!